Makefile

```diff
@@ -42,11 +42,18 @@ export FLAGS		 = -std=gnu99 \
 			   -lnosys \
 			   -Wl,-gc-sections \
 			   -Wl,-g \
-			   -Werror
+			   -Werror \
+			   -Xlinker -Map=$(BUILD_DIR_ROOT)/$@/output.map
```

Comment:
@jlaitine please also see this:
https://github.com/tiiuae/Bootloader/pull/1
If we are going to add the ".map" file for the bootloader, it is nicer if had a uniform name.

Reply:
Sorry, I missed it. I cherry picked you fix on top in this same PR, is this ok?